lib/libutee/include/tee_api_defines.h

```diff
@@ -79,6 +79,7 @@
 #define TEE_PARAM_TYPE_VALUE_INPUT      1
 #define TEE_PARAM_TYPE_VALUE_OUTPUT     2
 #define TEE_PARAM_TYPE_VALUE_INOUT      3
+#define TEE_PARAM_TYPE_MEMREF_SECURE    4
```

Comment:
This define wouldn't be needed at all if the shared memory object itself managed the secure state.